sql/2020/22_HTTP_2/adoption_of_http_2_by_site_and_requests.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 22_HTTP_2 - Adoption of HTTP/2 by site and requests
```

Comment:
Do you also want to distinguish between first and third party requests? The test for first party is if `NET.HOST(url) = NET.HOST(page)`.

Reply:
Hi. I understand what you are saying and think it makes good sense. Are you suggesting that I modify this one query or use this query as the basis for an extra query. Some SQL guidance would be helpful. Thanks.

Reply:
@dotjs what would be your preference?

Reply:
My 2 cents: Personally I’m not so sure.

The firstHTML is good to measure separately (do the pages support HTTP/2?), and interesting to see that X% of requests support HTTP/2 overall. Do I care the 1st party/third party breakout of those requests? Not so much for this chapter IMHO since we have the firstHTML one so know whether the site supports it or not. Plus that will exclude sharded asset domains anyway.

Then again could just query it and see if anything interesting shows up.

Reply:
@rviscomi - @bazzadp and I discussed this and the collective opinion is that this is not extremely useful information